web/tools/webpack.config.ts

```diff
@@ -13,6 +13,8 @@ import 'webpack-dev-server'
 import loadBuildConfig, { WEB } from 'build-configs'
 import { WebBuildConfigType } from 'build-configs/BuildConfigType'
 
+const TerserPlugin = require('terser-webpack-plugin')
```

Comment:
🔧 Please use `import` over `require` here. You probably need to add `terser-webpack-plugin` to `devDependencies` for this.